packages/runtime/src/runtime/modal.tsx

```diff
@@ -24,21 +31,61 @@ export const ModalWrapper: React.FC = () => {
   const [content, setContent] = useState<React.ReactNode>(null);
   const [options, setOptions] = useState<ModalProps>({});
   const [isFullScreen, setIsFullScreen] = useState(false);
+  const [modalWidth, setModalWidth] = useState(0);
+  const [modalHeight, setModalHeight] = useState(0);
+  const [key, setKey] = useState(0);
+
+  useLayoutEffect(() => {
+    if (visible && !isFullScreen) {
+      const contentNode = document.querySelector(".ant-modal-content");
+      setModalWidth(contentNode?.clientWidth || 0);
+      setModalHeight(contentNode?.clientHeight || 0);
+    }
+  }, [visible, isFullScreen, content]);
 
   const openModal = (
     content: React.ReactNode,
     newOptions: ModalProps,
   ): void => {
+    setKey((prevKey) => prevKey + 1);
```

Comment:
What's the purpose of the key?

Reply:
when a new modal is opened from within an already opened one, Modal title was not updating, so putting a unique key was the solution. Although we use `Math.random()` as well, what do you say?

Reply:
I see, so it's forcing a re-render here as I thought. I think this incrementing is fine